src/fundus/scraping/scraper.py

```diff
@@ -34,6 +34,7 @@ async def scrape(
             supported_attributes = set(
                 more_itertools.flatten(collection.names for collection in self.parser.attribute_mapping.values())
             )
+            supported_attributes.add("free_access")
```

Comment:
This should be removed and was only necessary because the attribute was excluded in the first place.

Reply:
It's weird I thought I needed to add it to keep the BaseParser attribute tests from failing, but now, after I removed it, everything still works fine